src/univariate/continuous/johnson.jl

```diff
@@ -0,0 +1,141 @@
+"""
+    Johnson(ξ, λ, γ, δ)
+
+The Johnson's ``S_U``-distribution with parameters ξ, λ, γ and δ is a transformation of the normal distribution:
+
+```math
+z \\sim \\gamma + \\delta \\sinh^{-1}\\Big(\\frac{x-\\xi}{\\lambda}\\Big),
+```
+
+where ``z \\sim \\mathcal{N}(0,1)``.
+
+```julia
+Johnson()           # Equivalent to Johnson(0.0, 1.0, 0.0, 1.0)
+Johnson(ξ, λ, γ, δ) # Johnson's S_U-distrubtion with shape parameters ξ, λ, γ and δ
+
+params(d)           # Get the parameters, i.e. (ξ, λ, γ, δ)
+shape(d)            # Get the shape parameter, i.e. ξ
+scale(d)            # Get the scale parameter, i.e. λ
+```
+
+External links
+
+* [Johnson's ``S_U``-distribution on Wikipedia](http://en.wikipedia.org/wiki/Johnson%27s_SU-distribution)
+"""
+struct Johnson{T<:Real} <: ContinuousUnivariateDistribution
+	ξ::T
+	λ::T
+	γ::T
+	δ::T
+	Johnson{T}(ξ, λ, γ, δ) where {T} = new{T}(ξ, λ, γ, δ)
```

Comment:
The indentation seems wrong? Did you use tabs? Please change it to four spaces throughout the files.
```suggestion
    ξ::T
    λ::T
    γ::T
    δ::T
    Johnson{T}(ξ::T, λ::T, γ::T, δ::T) where {T<:Real} = new{T}(ξ, λ, γ, δ)
```

Reply:
Yes the indentation were wrong. Now changed to spaces.